python/ray/scripts/scripts.py

```diff
@@ -2537,11 +2536,18 @@ def add_command_alias(command, name, hidden):
 cli.add_command(cpp)
 cli.add_command(disable_usage_stats)
 cli.add_command(enable_usage_stats)
-add_command_alias(job_cli_group, name="job", hidden=True)
 cli.add_command(state_cli_list)
 cli.add_command(state_cli_get)
 add_command_alias(summary_state_cli_group, name="summary", hidden=True)
 
+try:
+    from ray.dashboard.modules.job.cli import job_cli_group
```

Comment:
why do we do this specifically for this command?

Reply:
jobs cli depends on pydantic which is only included in with ray[default]. This is similar to serve cli below this line.